bedrock/mozorg/forms.py

```diff
@@ -51,7 +51,7 @@ def render(self, name, value, attrs=None):
         input_txt = super(PrivacyWidget, self).render(name, value, attrs)
 
         policy_txt = _(u'I’m okay with Mozilla handling my info as explained '
-                       u'in <a href="%s">this Privacy Notice</a>')
+                       u'in <a href="%s" style="color:#000">this Privacy Notice</a>')
```

Comment:
CSS styling should be applied to the [CSS file associated with this page](https://github.com/mozilla/bedrock/blob/master/media/css/newsletter/moznewsletter-subscribe.less#L223), not added directly to the HTML.

Adding the following style rule inside the `#newsletter-subscribe.light` selector should fix it:

```css
a:link,
a:visited {
    color: @linkBlue;
}
```